ShowerShow/Repository/Interface/IDisclaimersRepository.cs

```diff
@@ -0,0 +1,15 @@
+﻿using ShowerShow.Model;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ShowerShow.Repository.Interface
+{
+    public class IDisclaimerRepository
+    {
+        public Task<Disclaimers> GetDisclaimres(Disclaimers disclaimers);
```

Comment:
Everything dislaimer and term condition related should move to the other function